deeprankcore/trainer.py

```diff
@@ -541,13 +542,15 @@ def train( # pylint: disable=too-many-arguments, too-many-branches, too-many-loc
                         Defaults to False.
             num_workers (int, optional): How many subprocesses to use for data loading. 0 means that the data will be loaded in the main process.
                         Defaults to 0.
-            save_best_model (Optional[bool], optional): 
-                        If True, the best model (in terms of validation loss) is saved.
-                        If False, the last model tried is saved.
-                        If None, no model is saved.
+            best_model (bool, optional): 
+                        If True, the best model (in terms of validation loss) is selected for later testing or saving.
+                        If False, the last model tried is selected.
                         Defaults to True.
-            output_prefix (Optional[str], optional): Name under which the model is saved. A description of the model settings is appended to the prefix.
-                        Defaults to None.
+            save_model (bool, optional): 
+                        If True, the selected model is saved in <filename>.
+                        If False, no model is saved.
+                        Defaults to True.
+            filename (str, optional): Name of the file where to save the selected model. Defaults to 'model.pth.tar'.
```

Comment:
See comment above

```suggestion
            filename (Optional[str, optional]): Name of the file where to save the selected model. 
                ```Defaults to 'model.pth.tar'.
                If None, model is not saved
            best_model (bool, optional): 
                        If True, the best model (in terms of validation loss) is selected for later testing or saving.
                        If False, the last model tried is selected.
                        Defaults to True.
            save_model (bool, optional): 
                        If True, the selected model is saved in <filename>.
                        If False, no model is saved.
                        Defaults to True.
```